src/Cake.Common/Tools/OctopusDeploy/OctopusDeployAliases.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using Cake.Core;
+using Cake.Core.Annotations;
+
+namespace Cake.Common.Tools.OctopusDeploy
+{
+    /// <summary>
+    /// Contains functionality for working with Octopus Deploy.
+    /// </summary>
+    [CakeAliasCategory("Octo")]
```

Comment:
The category is used for the [documentation](http://cakebuild.net/dsl/), so something like `Octopus Deploy` would probably describe the aliases better.